app/models/league.rb

```diff
@@ -55,4 +55,8 @@ def round_robin_pairings(teams_array)
       [[teams_array.first, pivot]] + (1...(n / 2)).map { |j| [teams_array[j], teams_array[n - 1 - j]] }
     end
   end
+
+  def self.human_game_mode(mode)
+    I18n.t("activerecord.attributes.league.game_modes.#{mode}")
```

Comment:
Why did you change `game_modes` enum to singular, but use the plural in the translation key?

Reply:
Because rails magic. The enums actually had the *wrong name the whole time*. However, rails automatically creates a corresponding plural method that returns all enum constants. Since this is the set of all game modes, i decide to use the plural for the translation keys